Server/mods/deathmatch/logic/CResourceChecker.Data.h

```diff
@@ -324,106 +111,6 @@ namespace
-        {"cancelLatentEvent", "1.3.0-9.03772"},
-        {"getAllAccountData", "1.3.0-9.03804"},
-        {"reloadBans", "1.3.0-9.03931"},
-        {"getVehicleSirenParams", "1.3.0-9.03968"},
-        {"removeVehicleSirens", "1.3.0-9.03968"},
-        {"getVehicleSirens", "1.3.0-9.03968"},
-        {"addVehicleSirens", "1.3.0-9.03968"},
-        {"setVehicleSirens", "1.3.0-9.03968"},
-        {"setJetpackWeaponEnabled", "1.3.0-9.04351"},
-        {"getJetpackWeaponEnabled", "1.3.0-9.04351"},
-        {"sha256", "1.3.1-9.04835"},
-        {"setPedWalkingStyle", "1.3.1-9.04935"},
-        {"setAircraftMaxVelocity", "1.3.1-9.04997"},
-        {"getAircraftMaxVelocity", "1.3.1-9.04997"},
-        {"bitAnd", "1.3.2"},
-        {"bitNot", "1.3.2"},
-        {"bitOr", "1.3.2"},
-        {"bitTest", "1.3.2"},
-        {"bitXor", "1.3.2"},
-        {"bitLRotate", "1.3.2"},
-        {"bitRRotate", "1.3.2"},
-        {"bitLShift", "1.3.2"},
-        {"bitRShift", "1.3.2"},
-        {"bitArShift", "1.3.2"},
-        {"bitExtract", "1.3.2"},
-        {"bitReplace", "1.3.2"},
-        {"getPlayerACInfo", "1.3.3"},
-        {"addDebugHook", "1.3.4-9.05939"},
-        {"removeDebugHook", "1.3.4-9.05939"},
-        {"teaEncode", "1.3.5"},
-        {"teaDecode", "1.3.5"},
-        {"base64Encode", "1.3.5"},
-        {"base64Decode", "1.3.5"},
-        {"pregFind", "1.3.5"},
-        {"pregReplace", "1.3.5"},
-        {"pregMatch", "1.3.5"},
-        {"setElementCallPropagationEnabled", "1.3.5-9.06118"},
-        {"isElementCallPropagationEnabled", "1.3.5-9.06118"},
-        {"resendPlayerACInfo", "1.5.1-9.07633"},
-        {"dbPrepareString", "1.5.2"},
-        {"isPedWearingJetpack", "1.5.5-9.13846"},
-        {"setPedWearingJetpack", "1.5.5-9.13846"},
-        {"setElementAngularVelocity", "1.5.5-9.14060"},
-        {"getElementAngularVelocity", "1.5.5-9.14060"},
-
-        // Features added in 1.5.8
-        {"xmlLoadString", "1.5.7-9.19626"},
-        {"getPlayerScriptDebugLevel", "1.5.7-9.19626"},
-        {"setPlayerScriptDebugLevel", "1.5.7-9.19626"},
-        {"getRemoteRequests", "1.5.7-9.20307"},
-        {"getRemoteRequestInfo", "1.5.7-9.20307"},
-        {"abortRemoteRequest", "1.5.7-9.20307"},
-        {"addColPolygonPoint", "1.5.7-0.20397"},
-        {"getColPolygonPointPosition", "1.5.7-0.20397"},
-        {"getColPolygonPoints", "1.5.7-0.20397"},
-        {"getColShapeRadius", "1.5.7-0.20397"},
-        {"getColShapeSize", "1.5.7-0.20397"},
-        {"removeColPolygonPoint", "1.5.7-0.20397"},
-        {"setColPolygonPointPosition", "1.5.7-0.20397"},
-        {"setColShapeRadius", "1.5.7-0.20397"},
-        {"setColShapeSize", "1.5.7-0.20397"},
-        {"isResourceProtected", "1.5.7-9.20468"},
-        {"hasElementData", "1.5.7-9.20477"},
-        {"addElementDataSubscriber", "1.5.7-9.20477"},
-        {"hasElementDataSubscriber", "1.5.7-9.20477"},
-        {"removeElementDataSubscriber", "1.5.7-9.20477"},
-
         // Features added in 1.5.9
```

Comment:
1.5.9 additions should also be removed (as master is 1.6 now)

Reply:
> 1.5.9 additions should also be removed (as master is 1.6 now)

fixed